doc/code/qml_noise.rst

```diff
@@ -50,15 +50,15 @@ quantum circuit. One can construct standard Boolean functions using the followin
     ~wires_eq
     ~wires_in
 
-For example, a Boolean function that checks of an operation is on wire ``0`` can be created
-like so:
+For example, a Boolean function that checks if an operation is on wire ``0`` can be created
+as follows:
 
 >>> fn = wires_eq(0)
 >>> op1, op2 = qml.PauliX(0), qml.PauliX(1)
 >>> fn(op1)
-False
->>> fn(op2)
 True
+>>> fn(op2)
```

Comment:
```suggestion

>>> fn(op2)
```

I believe usually we start a new code block after a return

Reply:
I have all the missing spaces from that module, currently trying to fix the sphinx errors 😅